docs/community/index.rst

```diff
@@ -0,0 +1,9 @@
+Community
+===============
+
+.. toctree::
```

Comment:
Same as for integrations :

Should we use :hidden: here, and use cards instead ?

Reply:
done